mappers/mqtt-mapper/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.20.10-alpine3.18 AS builder
+
+WORKDIR /build
+
+ENV GO111MODULE=on \
+    GOPROXY=https://goproxy.cn,direct
+
+COPY . .
+
```

Comment:
remove this file

Reply:
Thanks for your advice. I will modify the file structure.